app/controllers/posts_controller.rb

```diff
@@ -2,11 +2,11 @@ class PostsController < ApplicationController
   def index
     @user = User.find params[:user_id]
     @posts = Post.where(author_id: params[:user_id])
-    @comments = Comment.all
+    @comments = Comment.all.includes(:author)
 
     @comments_info = []
     @comments.each do |comment|
-      user = User.find comment.author_id
+      user = comment.author
       @comments_info << {
         user_name: user.name,
         text: comment.text,
```

Comment:
- [ ] I think the N+1 problem as requested in the [requirements](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/intergration_spec_and_n_1_project.md),  could be solved like this: 
`@posts = @user.posts.includes(:comments)`, please kindly fix that.